lib/car.rb

```diff
@@ -0,0 +1,29 @@
+require 'pry'
+class Car
+  attr_reader :make, :model, :monthly_payment, :loan_length, :color
```

Comment:
Since you have defined methods for make and model below, it is not necessary to also add them to the attr_reader list. attr_reader essentially creates a hidden "getter" method for any attributes in the list. For example with :monthly_payment ruby creates this method for us:

```
def monthly_payment
  @monthly_payment
end
```

but because you have the methods already defined it doesn't make sense

```
  def make
    make = @make_model.split()
    return make.first
  end
```

```suggestion
  attr_reader :monthly_payment, :loan_length, :color
```